pkg/tnf/handlers/handler_template/template.tmpl

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2020-2021 Red Hat, Inc.
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+package {{ .Handlername }}
+
+import (
+	"time"
+
+	"github.com/test-network-function/test-network-function/pkg/tnf"
+	"github.com/test-network-function/test-network-function/pkg/tnf/dependencies"
+	"github.com/test-network-function/test-network-function/pkg/tnf/identifier"
+	"github.com/test-network-function/test-network-function/pkg/tnf/reel"
+)
+
+// Hostname provides a hostname test implemented using command line tool "hostname".
+type {{ .Handlername }} struct {
+	result  int
+	timeout time.Duration
+	args    []string
+	// The hostname
+	hostname string
+}
+
+const (
+	// Command is the command name for the unix "hostname" command.
```

Comment:
this is not repeated accross all the files.
If something is not common between all the tests, then it should be replaced with //TO_DO or a generic comment